src/site/content/en/fast/signed-exchanges-webpackager/index.md

```diff
@@ -382,86 +424,28 @@ used with signed exchanges.
+   site's CA certificate. More instructions on this can be found
+   [here](https://www.digicert.com/kb/ssl-support/pem-ssl-creation.html).
 
-1. Follow the steps 1 through 3 of [Generate a self-signed
-   certificate](/signed-exchanges-webpackager/#generate-a-self-signed-certificate).
-2. Convert `cert.pem` to the `application/cert-chain+cbor` format. 
-
-    ```shell
-    gen-certurl -pem cert.pem -ocsp &lt;(echo ocsp) > cert.cbor
-    ```
-
-    Certificates come in many formats. `cert.pem` is in the [PEM
-    format](https://en.wikipedia.org/wiki/Privacy-Enhanced_Mail). Certificates for
-    signed exchanges must be in the `application/cert-chain+cbor` format.
-
-    When using a self-signed certificate, `webpkgserver` automatically converts
-    the certificate indicated by the `PEMFile` option in `webpkgserver.toml` to
-    the `application/cert-chain+cbor` format. However, if you are using a
-    `CanSignHttpExchanges` certificate, you must generate the CBOR-encoded
-    certificate yourself.  
-
-
-### Setup up Web Packager for production use
-
-
-
-1. Follow steps 1 and 2 of [Setup the Web Packager Server for
-   testing](/signed-exchanges-webpackager/#setup-the-web-packager-server-for-testing).
+   [PEM](https://en.wikipedia.org/wiki/X.509#Certificate_filename_extensions) is
+   a file format that is commonly used as a "container" for storing multiple
+   certificates.
 
 2. Open `webpkgserver.toml` with the editor of your choice and make the
    following changes:
-    *   Change the line `PEMFile = cert.pem` to `PEMFile = cert.cbor`.
-
-
-### Upload the `CanSignHttpExchanges` certificate
-
-
-
-1.  Follow steps 1 through 4 of [Upload the self-signed
-    certificate](/signed-exchanges-webpackager/#upload-the-self-signed-certificate) to upload `cert.cbor`. 
-
-3. Adjust your server config to serve `cert.cbor` using the `Content-Type:
-   application/cert-chain+cbor` response header.
-
-    If this header is not set, you will see the following error when you inspect the
-    SXG in DevTools: `Content type of cert-url must be application/cert-chain+cbor.
-    Actual content type: text/html`.
-
-4. Restart `webpkgserver`.
-
-    ```shell
-    webpkgserver
-    ```
+    *   Update the line `PEMFile = cert.pem` to reflect the location of your PEM file.
+    *   (Optional) To have `webpkgserver` auto-renew the SXG certificate every
+        90 days, in the `[SXG.ACME]` section of `webpkgserver.toml`, change
+        `#Enable = false` to `Enable = true`. This option only applies to sites
+        with a DigiCert ACME account setup.
+
+3.  Configure your edge server to forward traffic to the `webpkgserver` instance
+    as well as serve the SXG certificate. For more information, see [Running behind
+    front-end edge
+    server](https://github.com/google/webpackager/blob/e8814274abbe55e8a8083fc01d2394aa7c077b8d/cmd/webpkgserver/README.md#running-behind-front-end-edge-server).
```

Comment:
maybe s/e8814274abbe55e8a8083fc01d2394aa7c077b8d/master/ so we can update the target documentation